conftest.py

```diff
@@ -5,6 +5,18 @@
 from cloudinit import subp
 
 
+def _closest_marker_args_or(request, marker_name: str, default):
```

Comment:
is this supposed to be `_closest_marker_args_or_default` ?

Reply:
I'd be happy to land such a change if you want, but (unsurprisingly, as I wrote it :grin:) I think its intent is clear (and means that callers can generally call it on a single LOC; though that isn't much of an advantage if this spelling isn't clear!).

If we do want to change it, `_get_closest_marker_args` might be better, following the underlying API (and `dict.get` which takes a default)?

Reply:
No need to change it. I think it's fine. I just wanted to make sure it was what you intended and not an accidental misnaming.

Reply:
Also, I would like to point out the irony of me pointing out a possible typo, while at the same time writing `the code look a lot nice.` :smile: